Vocable/Features/Settings/SettingsViewController.swift

```diff
@@ -52,6 +52,29 @@ final class SettingsViewController: VocableCollectionViewController, MFMailCompo
                 return ""
             }
         }
+        
+        var accessibiltyId: AccessibilityID {
+            switch self {
+            case .categories:
+                return .settings.categoriesAndPhrasesCell
+            case .timingSensitivity:
+                return .settings.timingAndSensitivityCell
+            case .resetAppSettings:
+                return .settings.resetAppSettingsCell
+            case .selectionMode:
+                return .settings.selectionModeCell
+            case .privacyPolicy:
+                return .settings.privacyPolicyCell
+            case .contactDevs:
+                return .settings.contactDevelopersCell
+            case .listeningMode:
+                return .settings.listeningModeCell
+            case .pidTuner:
+                return ""
+            case .versionNum:
+                return ""
+            }
```

Comment:
Instead of the specifying the cases here, you can just say the default is an empty string

```suggestion
            default:
                return ""
```

Reply:
Thanks, updated!